dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -171,6 +179,8 @@ const SiteSelectionForm = ({ filepath, data, uiSchema }: SiteSelectionForm) => {
       return;
     }
 
+    storeFormData(formData);
```

Comment:
In my opinion this should be inside the try section after the validation. I don't think we want to save invalid data.